javascript/chronometer.js

```diff
@@ -1,34 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime=0;
+    this.intervalId=null;
   }
 
   start(callback) {
-    // ... your code goes here
+    const intervalId = setInterval(() => {this.currentTime +=1}, 1000); 
   }
 
   getMinutes() {
-    // ... your code goes here
+    let numberOfMinutes = Math.floor(this.currentTime/60);
+    return numberOfMinutes
   }
 
   getSeconds() {
-    // ... your code goes here
+   let numberOfSeconds = this.currentTime % 60
+   return numberOfSeconds
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    let twoDigitNumber
+    if (value<10){return `0${value}`
+  }else {return `${value}`}
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId)
+    
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime=0;
   }
 
   split() {
-    // ... your code goes here
+    return `${numberOfMinutes}:${numberOfSeconds}`
```

Comment:
these two variables are not defined. this section is doable using only the class' methods (getMinutes, getSeconds and split). Take a look at the solution to see a way of doing it